gmagickpixel_methods.c

```diff
@@ -258,3 +258,125 @@ PHP_METHOD(gmagickpixel, setcolorvalue)
 	GMAGICK_CHAIN_METHOD;
 }
 /* }}} */
+
+/* {{{ proto float GmagickPixel::getColorValueQuantum(int color )
+	Gets the quantum color of the GmagickPixel.
+*/
+PHP_METHOD(gmagickpixel, getcolorvaluequantum)
+{
+	php_gmagickpixel_object *internp;
+	zend_long colorquantum;
+	double color_value_quantum = 0;
```

Comment:
This probably should be defined as 'Quantum' type. Graphicsmagick uses different types internally to represent the actual value depending on the depth, and the various ones are:

typedef unsigned char Quantum;  
typedef unsigned short Quantum;  
typedef unsigned int Quantum;  

In ImageMagick, the quantum values can also be floats, if HDRI is enabled. That doesn't appear to happen with Graphicsmagick.